extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -194,6 +194,7 @@ export class AmpLightboxGallery extends AMP.BaseElement {
       this.element.appendChild(this.container_);
       this.manager_.maybeInit();
       this.buildMask_();
+      this.registerAction('open', invocation => this.activate(invocation));
```

Comment:
also document in  https://github.com/ampproject/amphtml/blob/master/spec/amp-actions-and-events.md